packages/EasyStandard/src/Sniffs/Arrays/AlphabeticallySortedArrayKeysSniff.php

```diff
@@ -0,0 +1,351 @@
+    }
+
+    /**
+     * @param \PhpParser\Node\Expr\ArrayItem[] $items
+     *
+     * @return \PhpParser\Node\Expr\ArrayItem[]
+     */
+    private function getSortedItems(array $items): array
+    {
+        foreach ($items as $index => $arrayItem) {
+            if ($arrayItem->value instanceof Array_) {
+                /** @var \PhpParser\Node\Expr\ArrayItem[] $subItems */
+                $subItems = $arrayItem->value->items;
+                if (\count($subItems) > 1) {
+                    $arrayItem->value->items = $this->getSortedItems($subItems);
+                }
+
+                $items[$index] = $arrayItem;
+            }
+
+            if ($arrayItem->value instanceof MethodCall) {
+                /** @var \PhpParser\Node\Expr\MethodCall $value */
+                $value = $arrayItem->value;
+                foreach ($value->args as $argIndex => $argument) {
+                    if ($argument->value instanceof Array_) {
+                        /** @var \PhpParser\Node\Expr\ArrayItem[] $subItems */
+                        $subItems = $argument->value->items;
+                        if (\count($subItems) > 1) {
+                            $argument->value->items = $this->getSortedItems($subItems);
+                        }
+
+                        $value->args[$argIndex] = $argument;
+                    }
+                }
+
+                $items[$index] = $arrayItem;
+            }
+        }
+
+        if ($this->isAssociativeOnly($items) === false) {
+            return $items;
+        }
+
+        \usort($items, function (ArrayItem $firstItem, ArrayItem $secondItem): int {
+            $firstName = $this->getArrayKeyAsString($firstItem);
+            $secondName = $this->getArrayKeyAsString($secondItem);
+
+            return $firstName <=> $secondName;
+        });
+
+        return $items;
+    }
+
+    /**
+     * @param \PhpParser\Node\Expr\ArrayItem[] $items
+     *
+     * @return bool
+     */
+    private function isAssociativeOnly(array $items): bool
+    {
+        $isAssociative = 1;
+
+        foreach ($items as $arrayItem) {
+            $isAssociative &= $arrayItem->key !== null;
+        }
+
+        return (bool)$isAssociative;
+    }
+
+    private function refactor(Array_ $node): Array_
+    {
+        /** @var \PhpParser\Node\Expr\ArrayItem[] $items */
+        $items = $node->items;
+
+        if ($this->isAssociativeOnly($items)) {
+            $items = $this->getSortedItems($items);
+
+            if ($node->items !== $items) {
+                $node->items = $this->fixMultiLineOutput($items);
+                $node->setAttribute('isChanged', true);
+            }
+        }
+
+        return $node;
+    }
+
+    private function setStartIndent(File $phpcsFile, int $bracketOpenerPointer): void
+    {
+        $token = $phpcsFile->getTokens()[$bracketOpenerPointer];
+        $indentLevel = (int)\floor(($token['column'] - 1) / 4);
```

Comment:
Please, extract `4` to a constant with a meaningful name.

Reply:
Done.